packages/ibm-products/src/components/Datagrid/Datagrid.docs-page.js

```diff
@@ -13,6 +13,7 @@ import toolbarScreenshot from './storybook-assets/datagrid-actions-example.png';
 
 const DocsPage = () => (
   <StoryDocsPage
+    deprecationNotice={`\`Datagrid\` will be deprecated in the next major version and moving to an example based approach. For more information please refer to the [migration docs](https://github.com/carbon-design-system/tanstack-carbon).`}
```

Comment:
Lets change this verbiage (`in the next major`) because we are announcing deprecation beginning Nov 20. And maybe add that support will remain into the next major (with sev 1 and security vul support only in v3).

Reply:
```suggestion
    deprecationNotice={`\`Datagrid\` is deprecated and will be moving to an example-based approach. For more information, please refer to the [migration docs](https://github.com/carbon-design-system/tanstack-carbon).`}
```

Details on major versions and sev/security should be captured in the `tanstack-carbon` README.